docs/Bio4jAWSReleases.md

```diff
@@ -1,45 +1,79 @@
-
-There isn't any kind of fee required for using Bio4j in AWS, you will just be paying for the data transfer _(and obviously the instances, volumes, deployed by yourself)_
-If you have any doubt please refer to this section on **[Requester Pays Buckets](http://docs.aws.amazon.com/AmazonS3/latest/dev/RequesterPaysBuckets.html)**.
-
-We provide two different options:
-
-### Lite version
-
-This database weighs **~460 GB** and includes the following modules:
-
-- UniProt (both SwissProt and TrEMBL) 
-- Gene Ontology (GO)
-- NCBI taxonomy
-- Enzyme DB
-- UniProtGO
-- UniProtEnzymeDB
-- UniProtNCBITaxonomy
-- UniProtInteractions
-- UniProtIsoforms
-
-The database files are provided as a **tar** file in the following address:
-
-> s3://eu-west-1.releases.bio4j.com/2014_12_03/bio4j_lite.tar
-
-### Whole version
-
-This version includes all modules and weighs **~1.2 TB**. Apart from the aforementioned, the following modules are available:
-
-- UniRef
-- UniProtUniRef
-- GenInfo
-- NCBITaxonomyGenInfo
-
-The database files are provided as a **tar** file in the following address:
-
-> s3://eu-west-1.releases.bio4j.com/2014_12_03/bio4j_whole.tar
-
-### Source files for the import
-
-The **XML/TXT** files from the different data sources used for this specific import were downloaded on the date **12/03/2014** and can be retrieved from the following bucket folder _(also stored in a Requester Pays Bucket)_:
-
-> s3://eu-west-1.raw.bio4j.com/
+# Pre-built Bio4j releases in AWS
+
+> This is the recommended way of using bio4j-titan. For this you need a working Amazon Web Services (AWS) account.
+
+We offer two pre-imported bio4j-titan distributions:
+
+- **bio4j lite** the size of the binaries is approximately `500GB`. It includes the following modules:  
+    - UniProt (both SwissProt and TrEMBL)
+    - Gene Ontology (GO)
+    - NCBI taxonomy
+    - Enzyme DB
+    - UniProtGO
+    - UniProtEnzymeDB
+    - UniProtNCBITaxonomy
+    - UniProtInteractions
+    - UniProtIsoforms
+- **bio4j full** the size of the binaries is approximately `1.2 TB`. This version includes all modules:
+    - UniProt (both SwissProt and TrEMBL)
+    - Gene Ontology (GO)
+    - NCBI taxonomy
+    - Enzyme DB
+    - UniProtGO
+    - UniProtEnzymeDB
+    - UniProtNCBITaxonomy
+    - UniProtInteractions
+    - UniProtIsoforms
+    - UniRef
+    - UniProtUniRef
+    - GenInfo
+    - NCBITaxonomyGenInfo
+
+They are available from S3 through a [requester pays](http://docs.aws.amazon.com/AmazonS3/latest/dev/RequesterPaysBuckets.html) bucket `s3://eu-west-1.releases.bio4j.com/`, in the `eu-west-1` (Ireland) region. The object addresses are
+
+- **bio4j lite** `s3://eu-west-1.releases.bio4j.com/2014_12_03/bio4j_all_but_uniref_and_gi_index.tar`
+- **bio4j full** `s3://eu-west-1.releases.bio4j.com/2014_12_03/bio4j_all_plus_isoforms.tar`
+
+The way this is expected to be used is:
+
+1. launch an EC2 instance in the `eu-west-1` region
+2. download the binary files for either bio4j lite or bio4j full.
```

Comment:
I think this is the most tricky part. Probably it's worth to mention this command: https://github.com/aws/aws-cli/issues/797#issuecomment-113461007